src/JSInterop/Microsoft.JSInterop.JS/src/tslint.json

```diff
@@ -0,0 +1,14 @@
+{
```

Comment:
Copied this from SignalR's tslint. Filed https://github.com/aspnet/AspNetCore/issues/7197 to enable this because there's  a bunch of warnings that show up when turned on.